docs/source/tutorial/peft_integrations.md

```diff
@@ -141,10 +141,12 @@ output = model.generate(**inputs)
 print(tokenizer.decode(output_disabled[0], skip_special_tokens=True))
 ```
 
-To disable the adapter, call the [`~PeftModel.disable_adapter`] method.
+To disable the adapter, call the `disable_adapters` method.
```

Comment:
Is this the correct disable adapter method? It'd be nice to link to it if it is :)

```suggestion
To disable the adapter, call the [`~diffusers.loaders.PeftAdapterMixin.disable_adapters`] method.
```